build/MSBuildExtensions.targets

```diff
@@ -99,7 +99,7 @@
     <ItemGroup>
       <_NETStandardLibraryVersions Include="@(PackageDefinitions->'%(Version)')"
                                    Condition="%(PackageDefinitions.Name) == 'NetStandard.Library'" />
-      <_NETCorePlatformsImplicitPackageVersion Include="@(PackageDefinitions->'%(Version)')"
+      <_BundledNETCorePlatformsVersion Include="@(PackageDefinitions->'%(Version)')"
```

Comment:
Suggest that same naming convention as nearby: _NETCorePlatformsVersions

Reply:
Sorry I was not careful of the naming pattern. I corrected it. It should be good now. However, I realize I am missing "Package" in "BundledNETCorePlatformsPackageVersion" so I need to sync up in SDK. Here is the SDK change https://github.com/dotnet/sdk/pull/1935/files

Once, there is a SDK build, I will remove the WIP and update the SDK version

Reply:
There is a pattern of XLibraryVersions -> XPackageVersion -> BundledXPackageVersion

I think I caught all of them